src/progressbar/styled-components.js

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import {styled} from '../styles/index';
+export const Root = styled('div', props => {
+  return {
+    width: '100%',
+  };
+});
+
+export const Bar = styled('div', props => {
+  const {$theme, $value, $successValue} = props;
+  const {colors, sizing, borders} = $theme;
+  let background = 'linear-gradient(to right';
+  const fillColor = colors.primary400;
```

Comment:
same question here, can we use template literals?

Reply:
ok